terraso_backend/apps/story_map/migrations/0003_alter_storymap_title.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1.6 on 2023-02-13 20:05
```

Comment:
```suggestion
# Copyright © 2021-2023 Technology Matters
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU Affero General Public License as published
# by the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public License
# along with this program. If not, see https://www.gnu.org/licenses/.
#
# Generated by Django 4.1.6 on 2023-02-13 20:05
```